analysis/topeft_run2/run_analysis.py

```diff
@@ -447,7 +486,7 @@ def LoadJsonToSampleName(jsonFile, prefix):
             # 'resource_monitor': True,
             "resource_monitor": "measure",
             "resources_mode": "auto",
-            #'filepath': f'/tmp/{os.environ["USER"]}', ##Placeholder to comment out if you don't want to save wq-factory dirs in afs
+            #'filepath': f'/tmp/{os.environ["USER"]}-workers', ##Placeholder to comment out if you don't want to save wq-factory dirs in $HOME
```

Comment:
Should we make this the default setting so users don't fill up their home dir?

Reply:
I would be in favor to do so